source/client/renderer/GameRenderer.cpp

```diff
@@ -676,20 +674,22 @@ void GameRenderer::render(float f)
 		}
 	}
 
+	// @TODO: Move to its own function
 	std::stringstream debugText;
 	debugText << "ReMinecraftPE " << m_pMinecraft->getVersionString();
-	debugText << "\n" << m_shownFPS << " fps, " << m_shownChunkUpdates << " chunk updates";
+	debugText << "\n" << m_shownFPS << " fps (" << m_shownChunkUpdates << " chunk updates)";
 
 	if (m_pMinecraft->getOptions()->m_bDebugText)
 	{
 		if (m_pMinecraft->m_pLocalPlayer)
 		{
 			char posStr[96];
 			Vec3 pos = m_pMinecraft->m_pLocalPlayer->getPos(f);
-			sprintf(posStr, "%.2f, %.2f, %.2f", pos.x, pos.y, pos.z);
+			sprintf(posStr, "%.2f / %.2f / %.2f", pos.x, pos.y, pos.z);
 
-			debugText << "\npos: " << posStr;
-			debugText << "\nentities: " << m_pMinecraft->m_pLevel->m_entities.size();
+			debugText << "\nXYZ: " << posStr;
```

Comment:
Ima be real, I have no idea what "XYZ" is even based on, but I don't wanna do what actual MC did by making "x:", "y:", "z:" on their own lines with super exact float displays. I feel like having "XYZ" rather than "Position" is a little cleaner and clearer since it describes the numbers in the order they appear.